packages/medusa/src/services/__mocks__/cart.ts

```diff
@@ -369,14 +374,16 @@ export const CartServiceMock = {
       }
     }
 
-    if (providerId === "nono") {
-      return {
-        provider_id: "nono",
-        data: {
-          money_id: "fail",
-        },
-      }
+    // RW 03/08/2022
```

Comment:
As above - Typescript was complaining that "not all code paths return an object" so I've made the second `if` the default return.